jazz_slack-event-handler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "jazz_slack-event-handler",
+    "version": "0.0.1",
+    "description": "jazz slack event handler",
+    "main": "index.js",
+    "private": true,
+    "author": [],
+    "scripts": {
+        "test": "mocha"
+    },
+    "dependencies": {
+        "string-template": "*",
+        "request": "*",
+        "lodash": "*",
```

Comment:
i guess since only _.includes is used you can replace it with ES5 native function and remove this usage.